docker/Dockerfile.tcf-dev

```diff
@@ -79,7 +79,7 @@ RUN apt-get update \
 # Install setuptools, py-solc and web3 packages using pip because
 # these are not available in apt repository.
 # Install nose2 for running tests.
-RUN pip3 install --upgrade setuptools json-rpc py-solc web3 nose2
+RUN pip3 install --upgrade setuptools jsonrpc json-rpc py-solc web3 nose2
```

Comment:
I'm not sure why we need both json-rpc and jsonrpc. Docker based setup is working fine today with just json-rpc. Also json-rpc is a better maintained project with Python3 support.

Reply:
OK, I will try it again.  I get the error `ModuleNotFoundError: No module named 'jsonrpc.exceptions'` if `json-rpc` is not installed, so I cannot run Avalon.  I do not care which is better maintained, I just want a clean install.

Reply:
We need only json-rpc (https://pypi.org/project/json-rpc/). 
jsonrpc (https://pypi.org/project/jsonrpc/) is not required.